cmd/root.go

```diff
@@ -169,6 +169,8 @@ When this flag is not set, there is no limit.`)
 to close after receiving a TERM signal. The proxy will shut
 down when the number of open connections reaches 0 or when
 the maximum time has passed. Defaults to 0s.`)
+	cmd.PersistentFlags().StringVar(&c.conf.Host, "host", "https://alloydb.googleapis.com/v1beta",
```

Comment:
Let's make the field `APIEndpointURL` and the flag should probably be `admin-ap-endpoint`. What do you think?

cc @kurtisvg

Reply:
Let's make this `alloydbadmin` to match the Cloud SQL version.

Reply:
Sounds good. I assume you meant `admin-api-endpoint` so changed it to that.